BLSQ/iaso
# Polio-1354 connect stocks to details view

Remove mock data from details view and use API

Related JIRA tickets :  POLIO-1354

## Self proofreading checklist


## Changes

Remove mock data from details view and use API

## How to test
Choose a country + vaccine combination
- In the django admin
    - Add a Vaccine request form with vaccine arrival report(s) 
    - Add Outgoung stock movement
    - Add destruction report(s)
    - Add Incident report(s)
 Then go to Polio > Vaccine Module > Stock management > details for the country + vaccine combination: check the usable and unusable tabs:
- Usable should be filled with data from the arrival reports as IN
- Usable should be filled with data from outgoing stock and incident reports as OUT
- Unusable should be filled with data from outgoing stock and incidnet reports as IN
- Unusable should be filled with data from destruction as OUT

## Print screen / video

https://github.com/BLSQ/iaso/assets/38907762/5ba78daa-224f-4fe4-a5df-bf2c0bf1b96b



## Notes

merging into https://github.com/BLSQ/iaso/tree/POLIO-1335-vaccine-stock-management-backend-form-a-destruction-incident-api
